doc/DeveloperGuide.md

```diff
@@ -88,6 +88,33 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename Tag
+
+**MSS**
+
+1. User requests to list all existing tags
+2. AddressBook shows a list of existing tags
+3. User requests to rename a tag in the list
+4. AddressBook renames the tag
+5. Addressbook updates the name change onto the corresponding tag of each person in the Addressbook
+Use case ends.
+
+**Extensions**
+
+2a. The list is empty
```

Comment:
In use case extensions, the numbering is use to describe scenarios that can happen, preventing the user from going to the next step.
so, this is better off being 1a.